src/data/partners.ts

```diff
@@ -0,0 +1,165 @@
+// Todo: Fill with Partners links & images (logos) - what in case of the non-existing images/links?
```

Comment:
We are currently collecting Images and Links of partners.
Maybe we should leave the placeholders now and fill it later on when we have all of the data ready?
Let's create a separate task for it. Shall we?

Reply:
https://app.clickup.com/t/8698cqvh8